src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud/config.jelly

```diff
@@ -55,6 +55,10 @@
       <f:entry title="${%Container Cleanup Timeout (minutes)}" field="retentionTimeout">
         <f:textbox default="5"/>
       </f:entry>
+
+      <f:entry title="${%Requires permission}" field="usageRestricted">
```

Comment:
Missing inline help (Add `help-usageRestricted.html`).

Reply:
added help file but still in the advanced section.